git-hooks/pre-push

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Not used for now
+# remote="$1"
+# url="$2"
+
+# Create the logs directory if not exists. It is ignored by git anyway
+test -d logs || mkdir logs
+
+# Check if ansible lint is installed
+lint_version=$(ansible-lint --version 2>&1)
+echo "Check if ansible-lint is installed and accessible: $lint_version"
```

Comment:
I think you want this to only be echo'd if ansible-lint is not installed?

Reply:
Thanks, I will do it too!